db/migrate/20160113184103_create_posts_tags_join_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePostsTagsJoinTable < ActiveRecord::Migration
+  def change
+    create_join_table :posts, :tags do |t|
+      # t.index [:post_id, :tag_id]
+      # t.index [:tag_id, :post_id]
```

Comment:
Nie ma indexow, sa pola. Jedyne indexy jakie sa to:

```
   add_index "users", ["email"], name: "index_users_on_email", unique: true
   add_index "users", ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
```

Nic o tagach

Dziwne ze tak sie generuje, w takim razie dobrze by bylo to odkomentowac i uruchomic migracje jeszcze raz